service-jehlomat/src/main/kotlin/services/Mailer.kt

```diff
@@ -68,13 +74,13 @@ class Mailer: MailerService {
     }
 
     @Throws(MailjetException::class)
-    override fun sendRegistrationConfirmationEmail(organization: Organization) {
+    override fun sendOrganizationConfirmationEmail(organization: Organization) {
         val request = MailjetRequest(Emailv31.resource)
             .property(
                 Emailv31.MESSAGES, prepareBody(
                     3222927,
```

Comment:
mimo tento PR, ale tady je spatne ID templatu `3222927 ` = nutno vytvorit novy pro schvaleni organizace superadminem.

Reply:
Souhlas, doplním explicitní TODO. Kromě id templatu se musí ještě opravit posílání na superadmin email, který momentálně v aplikaci nemáme.

Reply:
Jelikoz je kod verejny, tak bych nacital email superadmina z DB nebo z konfigurace ?

Reply:
TODO přidány

Ohledně emailu souhlasím, rozhodně je třeba hodnotu načítat. Založil jsem na to subtask JH-77, osobně bych použil sysenv jako pro připojení do db. Nemyslím si, že máme těch properties tolik, aby se vyplatil nějaký číselník v db.